src/views/domain-page/domain-page-header-status-tag/domain-page-header-status-tag.tsx

```diff
@@ -0,0 +1,24 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+
+import request from '@/utils/request';
+import DomainStatusTag from '@/views/shared/domain-status-tag/domain-status-tag';
+
+import { type DomainInfo } from '../domain-page.types';
+
+import { type Props } from './domain-page-header-status-tag.types';
+
+export default function DomainPageHeaderStatusTag(props: Props) {
+  const { data: domainInfo } = useSuspenseQuery<DomainInfo>({
+    queryKey: ['describeDomain', props],
+    queryFn: () =>
+      request(`/api/domains/${props.domain}/${props.cluster}`).then((res) =>
```

Comment:
Is it a separate query? Should we already get this information on domain information?

Reply:
It's not a separate query. react-query caches results for each query, so there is only one API call being made, with the second one being fetched from cache.